consul/ext_pillar/install.sls

```diff
@@ -5,6 +5,8 @@
 #  - configure `extension_modules` to point to that directory
 #  - configure ext_pillar and consul for both minion/master
 #  - ensure the consul IP is correct
+{%- set user = 'root' %}
```

Comment:
How do these changes fit into this PR?

Reply:
Good catch here is wrong should be `consul`